cards/src/main/resources/cards/custom/group2/set_633/spell_fury_of_the_elements.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Fury of the Elements",
+  "baseManaCost": 6,
+  "type": "SPELL",
+  "heroClass": "JADE",
+  "rarity": "FREE",
+  "description": "Summon two different Elemental Spirits",
+  "condition": {
```

Comment:
Not sure if this is the appropriate condition. This will make the card unable to cast if you have all three distinct elementals. I don't think it should have a condition. Check the test.